source/icu.net/NativeMethods/NativeMethods.cs

```diff
@@ -81,7 +81,26 @@ private static string dlerror()
 			var ptr = _dlerror();
 			return Marshal.PtrToStringAnsi(ptr);
 		}
+		private const string MAC_LIBDL_NAME = "libdl.dylib";
 
+		[DllImport(MAC_LIBDL_NAME, SetLastError = true)]
+		private static extern IntPtr mac_dlopen(string file, int mode);
```

Comment:
So Mac actually method names prefixed with `mac_` ? I wouldn't have expected that.